datafusion/Cargo.toml

```diff
@@ -16,97 +16,98 @@
 # under the License.
 
 [package]
-name = "datafusion"
+authors = ["Apache Arrow <dev@arrow.apache.org>"]
```

Comment:
😭why are there so many meaningless changes?

Reply:
I believe I accidentally started auto sorting my Cargo.toml. Or maybe it was related to upgrading cargo / rust? I'm not sure to be honest - trying to figure out what caused the change.